modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/list/List.js

```diff
@@ -25,21 +26,29 @@ import FrontendDataSetContext from '../../FrontendDataSetContext';
 import Actions from '../../actions/Actions';
 import ImageRenderer from '../../cell_renderers/ImageRenderer';
 
-const List = ({items, schema}) => {
+const List = ({items, schema, viewsTitle}) => {
 	const {selectedItemsKey} = useContext(FrontendDataSetContext);
 
 	return items?.length ? (
-		<ClayList>
-			{items.map((item, index) => {
-				return (
-					<ListItem
-						item={item}
-						key={item[selectedItemsKey] || index}
-						schema={schema}
-					/>
-				);
-			})}
-		</ClayList>
+		<div className="data-views-list-sheet">
+			{viewsTitle && 	
+				<div className="data-views-list-header">
+					<h2 className="sheet-title">{viewsTitle}</h2>
+				</div>
+			}
+
```

Comment:
I think you can use the sheet components in @clayui/layout here. You may need to [inspect the source code](https://github.com/liferay/clay/blob/master/packages/clay-layout/src/Sheet.tsx) to see how to use it...